cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -43,24 +40,23 @@ OPTIONS
        --code-coverage
            Enable code coverage. This will analyze the line coverage of the
            source `.bal` files in the Ballerina package and add a section
-           to the testerina report with the code coverage details.
-           This feature is not supported with single file executions.
+           to the testerina report with the code coverage details. This feature is not supported with single file executions.
 
        --test-report
            Generate an HTML report containing the test results. Defaults to 'true'
            if code coverage is enabled.
 
        --debug
-           Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
+           Run tests in remote debugging mode only when the '--skip-tests' flag is not used.
 
        --observability-included
-           Package observability in the executable JAR file(s).
+           Include package observability in the executable JAR file(s).
```

Comment:
This may need a bit more elaboration. Doesn't really add anything that the flag itself doesn't convey. What exactly does it mean to have observability enabled?

Reply:
@nadundesilva

Reply:
There are a set of dependencies (Jars, Ballerina module imports & sytax tree json) which gets added to the final executable if this flag is provided. As of now this is the only difference that gets introduced when this flag is provided.

There is a **future plan** to add the compiler instrumentation only if the flag is provided as well (this will be a non breaking change as currently Observability cannot be used without this flag).

So what we expect from this flag is to package everything related to Observability so that it can be used (Reference doc on starting Prometheus - https://central.ballerina.io/ballerinax/prometheus). This flag was introduced as a requirement to reduce the Jar sizes as well as Code injections completely when Observability is not required.

Reply:
Updated the description.